src/main/java/com/github/tomakehurst/wiremock/common/NetworkAddressRange.java

```diff
@@ -32,6 +32,11 @@ public abstract class NetworkAddressRange {
       Pattern.compile(SINGLE_IP.pattern() + "-" + SINGLE_IP.pattern());
 
   public static NetworkAddressRange of(String value) {
+
+    if (value == null || value.isEmpty()) {
+      throw new InvalidInputException(Errors.single(17, value + " is not a valid network address"));
```

Comment:
Why 17 by the way?

Reply:
Just copying the existing code: https://github.com/wiremock/wiremock/blob/e4dbb74069af90696fc058b809f8fa6783f10e67/src/main/java/com/github/tomakehurst/wiremock/common/NetworkAddressRange.java#L154-L161

Reply:
Not sure there's much logic to error codes beyond "uniquely identifies the problem"

Reply:
Ok :) 

```suggestion
      throw new InvalidInputException(Errors.single(42, value + " is not a valid network address"));
```

Reply:
Nope, NVM. I will merge as is